packages/components/app/styles/components/text.scss

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// TEXT
+//
+
+// .hds-text {}
+
+// ALIGNMENT
+
+.hds-text--align-left {
```

Comment:
it strikes me that these could end up being tempting for consumers to use in other contexts even if we would prefer they not 🤔

Reply:
```suggestion
.hds-text-don-t-use-me-in-line--align-left {
```
:trollface:

Reply:
I would argue this is a danger with any class

Reply:
True!